server/app/views/admin/programs/ProgramStatusesView.java

```diff
@@ -321,9 +333,8 @@ private Modal makeStatusUpdateModal(
       Http.Request request,
       ProgramDefinition program,
       Form<ProgramStatusesForm> form,
-      boolean displayOnLoad) {
-    // TODO(#2752): If an email is already configured, add a warning that setting it to empty
-    // will clear any other localized text.
+      boolean displayOnLoad,
+      Optional<DomContent> extraContent) {
```

Comment:
`extraContent` This feels unnecessarily vague, If we make it `boolean showEmailDeletionWarning` it adds clarity and readability.

Reply:
Done. One thing to note is that adding a second boolean has a different tradeoff in that it increases the risk of mixing variable order up. That said, I think introducing an AutoValue for this is too heavyweight so I'm not going to pursue that approach.

Reply:
ack. That's not unique here though and there are better ways to address it than what we had if it's of concern.   

If the method can be made static we could use AutoBuilder!!